prereise/gather/griddata/hifld/data_process/transmission.py

```diff
@@ -538,12 +539,13 @@ def create_buses(lines):
     return buses
 
 
-def create_transformers(bus):
+def create_transformers(bus, lines, transformer_designs):
     """Add transformers between buses within the same substation. The assumed topology
     is that the highest-voltage bus in each substation is connected via a tansformer to
     every other voltage.
 
-    :param pandas.DataFrame bus: columns 'sub_id' and 'baseKV'
+    :param pandas.DataFrame bus: columns 'sub_id' and 'baseKV'.
+    :param pandas.DataFrame lines: columns 'from_bus_id', 'to_bus_id', and 'rateA'.
```

Comment:
entry for `transformer_designs` is missing

Reply:
Done.